app/bundles/CampaignBundle/Membership/MembershipManager.php

```diff
@@ -129,6 +131,12 @@ public function addContacts(ArrayCollection $contacts, Campaign $campaign, $isMa
         foreach ($contacts as $contact) {
             $this->advanceProgressBar();
 
+            $this->logger->debug(
+                'CAMPAIGN: Contacts: '.count($contacts),
+                $contacts->toArray()
```

Comment:
This may be massive as the Lead entity can contain over 30 properties which are collections of contact events. Can this be just an array of IDs? Something like
```suggestion
                array_map($contact => $contact->getId(), $contacts->toArray())
```